src/main/kotlin/lotto/view/InputView.kt

```diff
@@ -0,0 +1,16 @@
+package lotto.view
+
+object InputView {
+    private const val ASK_LOTTO_BUY_MONEY = "구입금액을 입력해주세요."
+    private const val ASK_WINNING_LOTTO_NUMBERS = "지난주 당첨 번호를 입력해주세요."
+
+    fun askLottoBuyMoney(): Int {
+        println(ASK_LOTTO_BUY_MONEY)
+        return readLine()!!.toInt()
```

Comment:
```suggestion
        return readln().toInt()
```
코틀린 1.6부터는 향상된 읽기 함수를 제공합니다!